sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -56,6 +56,12 @@
     @VisibleForTesting
     static final int CLEAR_BATCH_SIZE = 100;
 
+    /**
+     * Transmission interval minimum value.
+     */
+    @VisibleForTesting
+    static final int MINIMUM_TRANSMISSION_INTERVAL_IN_SECONDS = 3;
```

Comment:
Wasn't this constant declared somewhere already? I think those can be merged and moved to fitting MS...Constants module in Core. 
Correct me if im wrong